### ISSUE TITLE
indentationGuess throws false positives on intermediate blank lines

### ISSUE THREAD

**Issue by schorfES** [schorfES/node-lintspaces#29]
As @denis-sokolov reports, `indentationGuess` throws false positives on intermediate blank lines. 

``` javascript
foo(function(){
  if (foo) {
    // One is good for you
    doOne();

    // Two is not bad as well  <-- warning on this line
    doTwo();
  }
})
```
